erica/application/tax_declaration/tax_declaration_dto.py

```diff
@@ -6,16 +6,24 @@
 from erica.erica_legacy.request_processing.erica_input.v1.erica_input import FormDataEst, MetaDataEst
 
 
+class FormDataEstDto(FormDataEst, BaseDto):
+    pass
+
+
+class MetaDataEstDto(MetaDataEst, BaseDto):
+    pass
+
```

Comment:
Is this just for naming and camelcasing? I'm a bit confused honestly. :D If it is, maybe add a comment? (I suppose it's because we don't want to change V1?

Reply:
These need to also inherit from BaseDto. Otherwise, they won't be camelCased. As you pointed out, I do not want to change the FormDataEst and I thought that this would make more sense